packages/template-retail-react-app/app/constants.js

```diff
@@ -34,6 +34,12 @@ export const HOME_SHOP_PRODUCTS_LIMIT = 10
 export const CAT_MENU_DEFAULT_NAV_SSR_DEPTH = 1
 export const CAT_MENU_DEFAULT_ROOT_CATEGORY = 'root'
 
+// TBD: Is there a better way to pass the color scheme
+export const PRODUCT_TILE_BADGE_LABELS = [
+    {label: 'New', color: 'green'},
+    {label: 'Sale', color: 'yellow'}
```

Comment:
Nit picking! 

there is a "hidden" requirement that the label displayed in the UI must be exactly the same word in your property name, prefixed with `c_is`.

What if a project has a property name isn't UI friendly, for example, `c_isNewSpecialBestProduct` (not the perfect example). We may want to decouple the property name from the actual label.

```suggestion
    {propertyName: 'c_isSale', label: 'Sale', color: 'yellow'}
```